### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by RedwoodSec** [jayofelony/pwnagotchi#217]
**Describe the bug**
Sometimes pwnagotchi goes blind and cannot continue without reboot major inconvenience. 

**To Reproduce**
1. Use Pwnagotchi 
2. Pwnagotchi goes blind and demands manual reboot
3. Sadness 

**Expected behavior**
Auto check for blind stalling and automatic restart any neccesary services or processes to get it scanning again automagically 
**Screenshots**
Inconvenience 

**Environment (please complete the following information):**
 - latest
 - OS version
 - raspberry pi zero
 - Any additional hardware used

**Additional context**
Major pain in my anus

**Comment by jayofelony** [comment]
Yeah, cant know every issue sadly a pi has. But it would help if you had any logs when it happens.
